marconi-chain-index/src/Marconi/ChainIndex/Orphans.hs

```diff
@@ -292,3 +292,12 @@ decodeLedgerState =
 
 deriving newtype instance SQL.ToField SecurityParam
 deriving newtype instance SQL.FromField SecurityParam
+
+-- * Sqlite, longer tuples
+
+instance (ToField a, ToField b, ToField c, ToField d, ToField e, ToField f,
```

Comment:
We should probably not do that, and instead use a proper data type for doing the insertion. If you don't want to do it in this PR, can you add a TODO here, and create a ticket for that?

Reply:
Will try to find a nicer solution!

Reply:
@kayvank What is the difference between `Utxo` and `UtxoRow`. The could be joined if it weren't for the To/FromJSON instances, was that a requirement from somebody from outside?

Reply:
Otherwise, switched from a tuple to a list for the `ToRow` instance d49a91235f53592724d483c88f26fdbf1044d566